packages/nodes-base/nodes/Wait/Wait.node.ts

```diff
@@ -294,6 +302,29 @@ export class Wait extends Webhook {
 				default: 'timeInterval',
 				description: 'Determines the waiting mode to use before the workflow continues',
 			},
+			{
+				displayName: 'Authentication',
+				name: 'incomingAuthentication',
+				type: 'options',
+				options: [
+					{
+						name: 'Basic Auth',
+						value: 'basicAuth',
+					},
+					{
+						name: 'None',
+						value: 'none',
+					},
+				],
+				default: 'none',
```

Comment:
we are still duplicating a lot, and the style of this block isn't consistent with the one after this.
The issue with duplication like this is that every time we need to update something in one place, we need to update it in all the copies. And we don't always do that, so we end up with slightly different versions of the same code that later someone else has to clean up.

Reply:
I would argue that here it is justifiable as Wait node basically include both Form and Webhook node, and though they have similar properties they are completely different entity, trying too mach reuse code we are tying them together so changes in Webhook would require Form change, which is not what we want

Ideally we want some library of common properties and helpers to setup them, like [returnAllOrLimit](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/utils/descriptions.ts) and helpers to populate options and fixedCollection

Reply:
What we need is a better way to describe properties, that's type-safe, and allows code reuse without being unintuitive.
Right now we are battling between duplicating code or making code unreadable, and neither is great.